src/style/blocks/cards.scss

```diff
@@ -0,0 +1,78 @@
+@import "../utils/variables";
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+  width: 200px;
+  height: 406px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 0 16px;
+
+  &__image {
+    margin: 32px auto 40px;
+    width: 160px;
+    height: 134px;
+    background-image: url("images/imac.jpeg");
+    background-position: center;
+    background-size: 160px 134px;
+  }
+
+  &__title {
+    text-decoration: none;
+
+    color: $main-accent;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    font-weight: $font-normal;
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary;
+    align-self: flex-start;
+    margin-bottom: 16px;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 22px;
+  }
+
+  &__review {
+    font-weight: $font-normal;
+    font-size: 10px;
+    color: $main-accent;
+    text-align: right;
+  }
+
+  &__price-box {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__button {
+    text-transform: uppercase;
+    height: 40px;
+    width: 166px;
+    left: 16px;
+    bottom: 16px;
```

Comment:
1. for inline-elements you cant set height, it's unnecessary here
2. position is static - you don't need to set left and bottom